dbt_subprojects/daily_spellbook/models/xrpl/gas_xrpl_fees.sql

```diff
@@ -0,0 +1,188 @@
+                ROW(
+                    'base_fee',
+                    CASE 
+                        WHEN p.price_usd IS NOT NULL 
+                        THEN (CAST(fee AS DOUBLE) / 1000000.0) * p.price_usd 
+                        ELSE NULL 
+                    END
+                ) AS ROW(fee_type VARCHAR, amount DOUBLE)
+            )
+        ] AS tx_fee_breakdown_usd
+        ,ARRAY[
+            CAST(
+                ROW(
+                    'base_fee',
+                    CAST(fee AS VARCHAR)
+                ) AS ROW(fee_type VARCHAR, amount VARCHAR)
+            )
+        ] AS tx_fee_breakdown_raw
+        ,'rrrrrrrrrrrrrrrrrrrrrhoLvTp' AS tx_fee_currency
+        ,CAST(NULL AS VARCHAR) AS block_proposer
+        ,CASE 
+            WHEN JSON_EXTRACT_SCALAR(metadata, '$.TransactionIndex') IS NOT NULL THEN 
+                CAST(JSON_EXTRACT_SCALAR(metadata, '$.TransactionIndex') AS BIGINT)
+            ELSE NULL 
+        END AS tx_index
+        ,transaction_type AS tx_type
+        ,JSON_EXTRACT_SCALAR(metadata, '$.TransactionResult') AS transaction_result
+        ,sequence
+        ,1 AS evt_index
+        ,'fee' AS transfer_type
+        ,'native' AS token_standard
+        ,CAST(account AS VARCHAR) AS from_address
+        ,CAST(NULL AS VARCHAR) AS to_address
+        ,'rrrrrrrrrrrrrrrrrrrrrhoLvTp' AS issuer
+        ,p.price_usd
+        ,fee AS fee_drops
+        
+    FROM {{ source('xrpl', 'transactions') }} t
+    LEFT JOIN xrp_prices p ON (
+        DATE_TRUNC('minute', CAST(
+            PARSE_DATETIME(
+                REGEXP_REPLACE(t._ledger_close_time_human, ' UTC$', ''),
+                'yyyy-MMM-dd HH:mm:ss.SSSSSSSSS'
+            ) AS TIMESTAMP
+        )) = p.price_minute
+    )
+    WHERE t.transaction_type IN (
+        'Payment'
+        ,'PaymentChannelClaim'
+        ,'CheckCash'
+        ,'AMMDeposit' 
+        ,'AMMWithdraw'
+        ,'EscrowFinish'
+    )
+        AND TRY_CAST(t.fee AS DOUBLE) > 0
+        {% if is_incremental() %}
+        AND {{ incremental_predicate('CAST(PARSE_DATETIME(REGEXP_REPLACE(t._ledger_close_time_human, \' UTC$\', \'\'), \'yyyy-MMM-dd HH:mm:ss.SSSSSSSSS\') AS TIMESTAMP)') }}
+        {% endif %}
+)
+SELECT
+    blockchain
+    ,block_month
+    ,block_date
+    ,block_time
+    ,block_number
+    ,tx_hash
+    ,tx_from
+    ,tx_to
+    ,gas_price
+    ,gas_used
+    ,gas_limit
+    ,gas_limit_usage
+    ,max_fee_per_gas
+    ,priority_fee_per_gas
+    ,max_priority_fee_per_gas
+    ,base_fee_per_gas
+    ,currency_symbol
+    ,tx_fee
+    ,tx_fee_usd
+    ,tx_fee_raw
+    ,tx_fee_breakdown
+    ,tx_fee_breakdown_usd
+    ,tx_fee_breakdown_raw
+    ,tx_fee_currency
+    ,block_proposer
+    ,l1_fee
+    ,l1_gas_used
+    ,l1_gas_price
+    ,l1_fee_scalar
+    ,tx_index
+    ,tx_type
+    ,transaction_result
+    ,sequence
+    ,evt_index
+    ,transfer_type
+    ,token_standard
+    ,from_address
+    ,to_address
+    ,issuer
+    ,price_usd
+    ,fee_drops
```

Comment:
```suggestion
    *
```

Reply:
similar to other PR, when final CTE does the explicit column selection, think it's fine to keep clean here